frontend/src/containers/RecipePage/Recipe.js

```diff
@@ -20,13 +20,15 @@ class Recipe extends Component {
       duration: '',
       servings: '',
       photo: '',
+      tags: [],
     };
   }
 
 
   componentDidMount() {
     window.scrollTo(0, 0);
     const { match: { params } } = this.props;
+    const { match: { params: { id } } } = this.props;
```

Comment:
It looks like we're doing this twice, in both cases to access `id`. We should probably just do one or the other (either extract `id` or `params.id`), and use it wherever we need it.